dumper/app/src/main/resources/oracle-stats/cdb/native/db-objects-synonym-public.sql

```diff
@@ -13,44 +13,26 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 SELECT
-  B.con_id "ConId",
+  A.con_id "ConId",
   'PUBLIC' "Owner",
   'SYNONYM' "ObjectType",
-  B.editionable "Editionable",
-  B.object_name "ObjectName",
-  -- This looks similar to filtering with WHERE and using count() instead of sum().
-  --
-  -- It is not similar. DB will see the LIKE inside a WHERE predicate and decide to
-  -- replace a HASH JOIN with NESTED LOOPS. The JOIN arguments have >10k rows each,
-  -- so performance-wise the nested loop would be terrible.
-  sum(
-    CASE WHEN B.object_name LIKE '/%' THEN 0
-    WHEN B.object_name LIKE 'BIN$%' THEN 0
-    ELSE 1 END
-  ) "Count"
-FROM (
-  SELECT
-    A.con_id,
-    A.editionable,
-    A.object_name,
-    A.owner
-  FROM cdb_objects A
-  WHERE A.object_type = 'SYNONYM'
-    AND A.owner = 'PUBLIC'
-) B
-LEFT JOIN (
-  SELECT
-    C.synonym_name,
-    C.con_id,
-    C.table_owner
-  FROM cdb_synonyms C
-  WHERE C.owner = 'PUBLIC'
-    AND C.table_owner IS NOT NULL
-) D ON B.object_name = D.synonym_name
-  AND B.con_id = D.con_id
-WHERE D.table_owner IS NULL
-    AND B.owner = 'PUBLIC'
+  A.editionable "Editionable",
+  A.object_name "ObjectName",
+  -- "Count" is kept for backwards compatibility
+  CASE WHEN A.object_name LIKE '/%' THEN 0
```

Comment:
I suggest to remove Count column from the synonyms sql. We can easily add this filtering on the assessment side. This will reduce dumper complexity.

Reply:
This is a good idea, but it must wait until Assessment no longer relies on this column. Even though the information in the column can be obtained elsewhere, current-version Assessment could still try to read it and get errors.

Reply:
How about we handle this condition in the same way as it is in db-objects.sql SQL and add `WHERE A.object_name NOT LIKE 'BIN$%' AND A.object_name NOT LIKE '/%'`? We can then keep simple `count(1)` for backward compatibility and remove it later after changing assessment.

Reply:
Done, I used the constant `1` instead of counting the items, because without grouping by name the number won't be useful.